ingress/controllers/nginx/nginx.tmpl

```diff
@@ -17,8 +17,14 @@ events {
 
 http {
     {{/* we use the value of the header X-Forwarded-For to be able to use the geo_ip module */}}
+    {{ if $cfg.useProxyProtocol -}}
+    set_real_ip_from    {{ $cfg.proxyRealIpCidr }};
+    real_ip_header      proxy_protocol;
```

Comment:
for my own eductaion, this actually sets the ip on the forwarded packet from the proxy protocol ascii block, right?